website/source/docs/enterprise/vcs/bitbucket.html.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: "enterprise"
+page_title: "BitBucket Cloud - VCS Integrations - Terraform Enterprise"
+sidebar_current: "docs-enterprise-vcs-bitbucket-cloud"
+description: |-
+  BitBucket Cloud repositories can be integrated with Terraform Enterprise by using push command.
+---
+# BitBucket Cloud
+
+BitBucket Cloud can be used to import Terraform configuration, automatically
```

Comment:
@kerrizor this is consistent with the way `configuration` is used in the original GitHub documentation.